scripts/setup-e2e.sh

```diff
@@ -88,10 +88,15 @@ if ! yarn build; then
   exit ${FAILED_SETUP}
 fi
 
+pushd ./dist
+  npx yalc publish
```

Comment:
can these calls to `npx yalc` be replaced with `yarn` commands?

Reply:
Replaced with `yarn test:apps:prepare`